Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -369,5 +369,13 @@ public override void OnInputUp(InputEventData eventData)
             // Poke pointer should not respond when a button is released or hand is un-pinched
             // It should only dispatch events based on collision with touchables.
         }
+
+        protected override void OnEnable()
+        {
+            base.OnEnable();
+
+            Debug.Assert(line != null, "No line renderer found in PokePointer.");
```

Comment:
Why does pokepointer need a line renderer? Not sure we want to assert here.

Reply:
No idea. My priority was just to move code out of OnValidate.